### ISSUE TITLE
[Info needed] Icon specs?

### ISSUE THREAD

**Issue by domvelo** [ionic-team/ionicons#763]
Hi there, 

I'm wondering, is there a icon design specification of dimensions?
I see that all icons have the size 512x512px in SVG format.
Then I checked out the icons with a circle like "checkmark-circle". The circle itself is 416x416px. This means 48px margin. But other icons overflows 416x416px. And also does not contain a circle of 512x512px. For example the "female" icon. It does not contain a 512px circle is not inside the margin circle (416px).

Here is a demo of the dimensions. Red edges indicates that the rectange of 416x416 can overflow the 512px circle. The inner circle of 416px is an exampe of the current circle icons the have. I thought this is the safe area and all my icons have to fit-contains it.

Indeed it depends of "How is the svg displayed at runtime." For example how is a svg displayed in a circle button? Is there no specification? Some icons like the "cloud-done" icon has no margin (padding) on the edges left and right. etc. The cloud icon width is 512.1px.
I think my new icon should not overflow the 416px rectangle and not the 512px circle to keep margin / padding. Or is this a rule of thumb? By watching the preview tests. Or golden section / cut / ratio?
In other words: I should just looking good. There is no limitations.

![app-icon](https://user-images.githubusercontent.com/59838794/72748066-964df700-3bb6-11ea-9b19-1cf1b4ce4304.png)

![ios-checkmark-circle](https://user-images.githubusercontent.com/59838794/72748987-0fe6e480-3bb9-11ea-8b18-f257547883b1.png)
![ios-cloud-done](https://user-images.githubusercontent.com/59838794/72748988-0fe6e480-3bb9-11ea-8883-2213e8e2f155.png)
![ios-female](https://user-images.githubusercontent.com/59838794/72748989-107f7b00-3bb9-11ea-9c4a-8508e529517c.png)

**Comment by brandyscarney** [comment]
Thanks for the issue! We do not currently have a design document with specifications for the icons and do not have plans to add this so I will be closing this.
